File: Greek For Greek (Python, JavaScript)/Print Linked List/Print Linked List.js
// You are given the head of a singly linked list. Return an array containing the values of the nodes.

/*
class Node {
  constructor(x){
    this.data = x;
    this.next = null;
  }
}

/**
 * @param {Node} head
 */
class Solution {
    printList(head) {
        // code here
        let arr = [];
        let temp = head;
        while (temp !== null){
            arr.push(temp.data);
            temp = temp.next;
        }
        return arr;
    }
}

File: Greek For Greek (Python, JavaScript)/Print Linked List/Print Linked List.py
# You are given the head of a singly linked list. Return an array containing the values of the nodes.

'''
class Node:
    def __init__(self, x):
        self.data = x
        self.next = None
'''

class Solution:
    def printList(self, head):
        # code here
        li = []
        temp = head
        while temp is not None:
            li.append(temp.data)
            temp = temp.next
        return li